logic/hosts.go

```diff
@@ -160,6 +160,9 @@ func UpdateHostFromClient(newHost, currHost *models.Host) (sendPeerUpdate bool)
 		currHost.EndpointIP = newHost.EndpointIP
 		sendPeerUpdate = true
 	}
+	if newHostPass, err := bcrypt.GenerateFromPassword([]byte(newHost.HostPass), 5); err == nil && string(newHostPass) != currHost.HostPass {
+		currHost.HostPass = string(newHostPass)
+	}
```

Comment:
but why would the password get updated for a host?

Reply:
cc @mattkasun

Reply:
I can't think of a use case for updating the Host password.  If it is required, this implementation is not complete in that the mq dynamic-security is not being updated.